applications/GeoMechanicsApplication/tests/test_rotation_with_moving_load.py

```diff
@@ -28,5 +30,33 @@ def test_rotation_with_moving_load(self):
         # updated, the results will be different.
         self.assertAlmostEqual(-0.000177858, value_for_node[2])
 
+    def test_rotation_with_moving_load_constant_system_matrices(self):
+        """
+        This test is the same as test_rotation_with_moving_load, but the system matrices are indicated as constant in
+        the solver settings. Therefore, another builder and solver is used.
+        """
+        test_name = "test_rotation_with_moving_load_constant_system_matrices"
+        file_path = test_helper.get_file_path(test_name)
+
+        # run simulation
+        simulation = test_helper.run_kratos(file_path)
+
+        self.assertTrue(isinstance(simulation._GetSolver().builder_and_solver, KratosGeo.ResidualBasedBlockBuilderAndSolverWithMassAndDamping))
+
+        res_path = os.path.join(file_path, test_name + result_extension)
```

Comment:
To get this variable declaration and initialization closer to its usage, we could move it to line 49.

Reply:
Agreed, moved it to where it's used